Integrations/MimecastV2/mimecast_test.py

```diff
@@ -0,0 +1,57 @@
+''' IMPORTS '''
+from __future__ import print_function
+import demistomock as demisto
+import pytest
+import MimecastV2
+
+# Parameters for Get arguments test
+policy_data = {
+        'description': 'new',
+        'fromPart': 'bla bla',
+        'fromType': 'free_mail_domains',
+        'fromValue': 'gmail.com',
+        'toType': 'email_domain',
+        'toValue': 'gmail.com',
+        'option': 'no_action',
+        'policy_id': 'IDFROMMIMECAST'
+    }
+
+policy_args = policy_data.copy()
+del policy_args['option']
+del policy_args['policy_id']
```

Comment:
why?

Reply:
Its need to be separate from the policy object

Reply:
so lets create another dict for it

Reply:
done